stylelint.config.js

```diff
@@ -210,7 +224,7 @@ module.exports = {
 		},
 		{
 			/* Validate that the legacy themes don't introduce any new selectors or custom properties */
-			files: ["components/*/themes/*.css", "!components/*/themes/spectrum.css"],
+			files: ["components/*/themes/express.css", "components/*/themes/spectrum-two.css"],
```

Comment:
```suggestion
			files: ["components/*/themes/express.css", "components/*/themes/spectrum.css"],
```

Ooops this was probably supposed to be !spectrum-two.css. 🙃 #bugfix